onnxruntime/core/framework/session_state.cc

```diff
@@ -1052,12 +1052,14 @@ Status SessionState::FinalizeSessionStateImpl(const std::basic_string<PATH_CHAR_
 
   ORT_RETURN_IF_ERROR(CreateKernels(kernel_registry_manager));
 
+#ifndef ENABLE_TRAINING
   const auto disable_prepacking =
       session_options.GetConfigOrDefault(kOrtSessionOptionsConfigDisablePrepacking, "0");
 
   if (disable_prepacking != "1") {
     ORT_RETURN_IF_ERROR(PrepackConstantInitializedTensors(constant_initializers_use_count));
   }
```

Comment:
Seems disable prepacking is off by default? If I read the code correctly, the change here just removed disable prepacking option, instead of changing the default value?

Reply:
PrepackConstantInitializedTensors is the function to prepack the tensor

Reply:
Maybe some comments why prepack is disabled for training here?